app.py

```diff
@@ -36,12 +37,24 @@
 logger.setLevel(config["log_level"])
 
 
+def dummy():
+    """
+    The sole purpose is to force Chalice to generate the right permissions in the policy.
+    Does nothing and returns nothing.
+    """
+    sqs = boto3.client("sqs")
+    sqs.send_message()
+    sqs.get_queue_url()
+
+
 @app.route(
     "/interactive",
     methods=["POST"],
     content_types=["application/x-www-form-urlencoded"],
 )
 def interactive():
+    if False:
```

Comment:
Why have this condition. When won't it be `False`?

Reply:
To get chalice to parse the function...